apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -939,11 +943,54 @@ with_log(Fun, Msg) ->
 cluster_info() ->
     RunningNodes = safe_call_mria(running_nodes, [], []),
     StoppedNodes = safe_call_mria(cluster_nodes, [stopped], []),
-    #{
+    Check = check_static_seeds(RunningNodes, StoppedNodes),
+    Check#{
         running_nodes => RunningNodes,
         stopped_nodes => StoppedNodes
     }.
 
+check_static_seeds(RunningNodes, StoppedNodes) ->
+    check_static_seeds(emqx_conf:get([cluster]), RunningNodes, StoppedNodes).
+
+check_static_seeds(
+    #{discovery_strategy := static, static := #{seeds := Seeds}},
+    RunningNodes,
+    StoppedNodes
+) ->
+    Msg0 = #{
+        seeds => Seeds,
+        discovery_strategy => static,
+        running_nodes => RunningNodes,
+        stopped_nodes => StoppedNodes
+    },
+    AllNodes = RunningNodes ++ StoppedNodes,
+    NodesMissInCluster = Seeds -- AllNodes,
+    NodeMissInSeeds = AllNodes -- Seeds,
+    case {NodesMissInCluster, NodeMissInSeeds} of
+        {[], []} ->
+            #{};
+        {[_ | _], _} ->
+            %% User should remove orphan nodes from seeds manually or start the orphan nodes.
+            %% so we call those orphan_nodes as `orphan_nodes_in_seeds`.
+            Msg = Msg0#{
+                msg => "node_configured_in_seeds_but_missing_in_cluster",
+                orphan_nodes_in_seeds => NodesMissInCluster
+            },
+            ?SLOG(warning, Msg),
+            Msg;
+        {_, [_ | _]} ->
+            %% User should add orphan nodes into seeds manually or stop the orphan nodes.
+            %% so we call those orphan_nodes as `orphan_nodes_not_in_seeds`.
+            Msg = Msg0#{
+                msg => "node_in_cluster_but_not_configured_in_seeds",
```

Comment:
I think this warning message is not necessary. One could configure one node as the seed and use it alone to bootstrap a cluster with however many nodes. It's not necessary to list all of the nodes as seeds: once the node connects to the cluster, the information about the rest of the nodes will be exchanged through other means.